packages/components/src/internal/util/messaging.spec.tsx

```diff
@@ -20,31 +20,31 @@ describe('resolveErrorMessage', () => {
+                'ERROR: duplicate key value violates unique constraint "uq_material_lsid"\n  Detail: Key (lsid)=(urn:lsid:labkey.com:Sample.252.Examples:E-20200121-743, E-20200121-743) already exists.\n  Where: SQL statement "INSERT INTO exp.material (lsid, name, cpastype, runid, created, container, createdby, modifiedby, modified, lastindexed, description, objectid)\nSELECT $1.lsid, $1.Name, \'urn:lsid:labkey.com:SampleSet.Folder-252:Examples\', $1.RunId, CURRENT_TIMESTAMP, \'578a3330-dee0-1037-bdca-4e2bf136af84\', 1005, 1005, CURRENT_TIMESTAMP, $1.LastIndexed, $1.Description, _$objectid$_\nRETURNING rowid"\nPL/pgSQL function temp.fn_cb4838bd5008e21ca0ebb3e8d366d2cf(temp.fn_cb4838bd5008e21ca0ebb3e8d366d2cftype) line 10 at SQL statement',
             extraContext: {},
             success: false,
             errors: [
                 {
                     exception:
-                        'ERROR: duplicate key value violates unique constraint "uq_material_lsid"\n  Detail: Key (lsid)=(urn:lsid:labkey.com:Sample.252.Examples:E-20200121-743) already exists.\n  Where: SQL statement "INSERT INTO exp.material (lsid, name, cpastype, runid, created, container, createdby, modifiedby, modified, lastindexed, description, objectid)\nSELECT $1.lsid, $1.Name, \'urn:lsid:labkey.com:SampleSet.Folder-252:Examples\', $1.RunId, CURRENT_TIMESTAMP, \'578a3330-dee0-1037-bdca-4e2bf136af84\', 1005, 1005, CURRENT_TIMESTAMP, $1.LastIndexed, $1.Description, _$objectid$_\nRETURNING rowid"\nPL/pgSQL function temp.fn_cb4838bd5008e21ca0ebb3e8d366d2cf(temp.fn_cb4838bd5008e21ca0ebb3e8d366d2cftype) line 10 at SQL statement',
+                        'ERROR: duplicate key value violates unique constraint "uq_material_lsid"\n  Detail: Key (lsid)=(urn:lsid:labkey.com:Sample.252.Examples:E-20200121-743, E-20200121-743) already exists.\n  Where: SQL statement "INSERT INTO exp.material (lsid, name, cpastype, runid, created, container, createdby, modifiedby, modified, lastindexed, description, objectid)\nSELECT $1.lsid, $1.Name, \'urn:lsid:labkey.com:SampleSet.Folder-252:Examples\', $1.RunId, CURRENT_TIMESTAMP, \'578a3330-dee0-1037-bdca-4e2bf136af84\', 1005, 1005, CURRENT_TIMESTAMP, $1.LastIndexed, $1.Description, _$objectid$_\nRETURNING rowid"\nPL/pgSQL function temp.fn_cb4838bd5008e21ca0ebb3e8d366d2cf(temp.fn_cb4838bd5008e21ca0ebb3e8d366d2cftype) line 10 at SQL statement',
```

Comment:
With the switch of lsid from using name to id, this type of error should no longer be present

Reply:
The exact text is different. I was lazy. Updated.